### DISCUSSION TITLE
Sequential Feature Selection

### DISCUSSION THREAD

**Discussion by tandamwa** [rasbt/mlxtend#1002] [Q&A]
Dear Prof. Raschka

Thank you for your contribution to the world through online content.

Just an observation, I have implemented a forward sequential feature selection, and the model is working properly with an accuracy of 98%. On each run, the model seems to select different features. Is this OK or is something wrong?

Thank you.

**Comment by NimaSarajpoor** [comment]
I was wandering and saw your question. When you say in each run, what exactly do you mean by that? I mean... do you consider the whole process of feature_selection as one run? Also, would it be possible for you to upload the code so that I can check it out?

**Comment by rasbt** [comment]
Hey @tandamwa. There may be a few sources of randomness where it is normal that you get different results for the different runs.
Does your ML model have a `random_state` parameter? If yes, I recommend setting it to a fixed integer value

**Reply by tandamwa** [reply]
Thank you @rasbt. I will try that.
